NDTensors/src/lib/Sectors/src/sector.jl

```diff
@@ -0,0 +1,109 @@
+const default_sector_size = 4
+default_sector_data() = Label
+const SStorage = SmallSet{default_sector_size,default_sector_data()}
+const MSStorage = MSmallSet{default_sector_size,default_sector_data()}
```

Comment:
We should discuss this code, I think there is a better code pattern for accomplishing the same thing without defining globals like this.

Reply:
Definitely we should avoid `const default_sector_size = 4` since that is then being used as a type parameter which is not good for type stability.